### DISCUSSION TITLE
What's "Exit code 0xc0000135"

### DISCUSSION THREAD

**Discussion by Naville** [microsoft/vcpkg#25773] [Q&A]
# Steps
- ``vcpkg install z3:x64-windows``
- ``vcpkg export z3:x64-windows``
- ``cmake -DXXXXXX -DCMAKE_TOOLCHAIN_FILE=xxx -DVCPKG_TARGET_TRIPLET="x64-windows" -DCMAKE_MSVC_RUNTIME_LIBRARY="MultiThreadedDLL"``

However find_package broke with:
```
-- Could NOT find Z3: Found unsuitable version "Exit code 0xc0000135
", but required is at least "4.7.1" (found E:/vcpkg_sysroot/x64-windows/lib/libz3.lib)
CMake Error at CMakeLists.txt:359 (message):
```

I used ``--debug-trycompile`` to preserve the try_compile vcproject, and then trying to run the compiled exe results in the following:

<img width="437" alt="image" src="https://user-images.githubusercontent.com/5205699/178936943-0f79affb-e11c-41cf-ab2e-ce642ed8a634.png">

[FindZ3.cmake](https://github.com/llvm/llvm-project/blob/release/9.x/llvm/cmake/modules/FindZ3.cmake)

I'm not familiar with Windows, but I can't use x64-windows-static due to my other dependencies don't support it

**Comment by JackBoosY** [comment]
Maybe a port bug, please fill an issue instead.

**Reply by Naville** [reply]
A user error due to I didn't add vcpkg's libz3.dll into %Path : )

**Reply by JackBoosY** [reply]
You don't need to add the path, it should be set by vcpkg automatically.
